lib/ui/screens/event_screen.dart

```diff
@@ -918,15 +953,13 @@ class _EventScreenState extends State<EventScreen> {
             state.result == null) {
           return Scaffold(
             appBar: ThaliaAppBar(title: const Text('EVENT')),
-            body: const Center(
-              child: CircularProgressIndicator(),
-            ),
+            body: const Center(child: CircularProgressIndicator()),
           );
         } else {
           final event = (state.result ?? widget.event)!;
           return Scaffold(
             appBar: ThaliaAppBar(
-              title: Text(event.title.toUpperCase()),
```

Comment:
Personally I find it better to keep this as the name or maybe prefix it with EVENT:, but leave the name of the event in, this is useful when an event has a longer description or a lot of members and you forgot the name whilst reading the event information.

Reply:
This has been decided with available technicie members to revert